test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,77 @@
+require('dotenv').config();
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+
+const urlBase = 'https://api.github.com';
+const githubUserName = 'Ktagudelo';
+const repositories = 'repos';
+const repositoryName = 'Proyecto-marvel';
+
+describe('GitHub User Create And Modify Issue With A Title', () => {
+  it('Consume GET GitHub User Logged', async () => {
+    const response = await axios.get(`${urlBase}/user`, {
+      headers: {
+        Authorization: `token ${process.env.ACCESS_TOKEN}`
+      }
+    });
+
+    if (response.data.public_repos > 1) {
+      expect(response.status).to.equal(StatusCodes.OK);
+    }
+  });
+
+  it('Consume GET GitHub User Repositories list', async () => {
+    const response = await axios.get(
+      `${urlBase}/users/${githubUserName}/${repositories}`
+    );
+    const reposUser = response.data.find(
+      (element) => element.name === 'Proyecto-marvel'
+    );
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(reposUser.owner.login).to.equal(githubUserName);
+    expect(reposUser.name).to.equal('Proyecto-marvel');
+  });
+
+  it('Consume POST GitHub User Create Issue', async () => {
+    const responseIssue = await axios.post(
```

Comment:
you can convert this `responseIssue` in a global variable or initialize it in a before statement, so you can reuse the `responseIssue.data` to get the issue id or number to make the patch call